tests/kamereon/test_kamereon_vehicles.py

```diff
@@ -125,9 +125,7 @@ def test_vehicles_response(filename: str) -> None:
                 "reports_charging_power_in_watts": vehicle_details.reports_charging_power_in_watts(),  # noqa: B950
                 "uses_electricity": vehicle_details.uses_electricity(),
                 "uses_fuel": vehicle_details.uses_fuel(),
-                "supports-hvac-status": vehicle_details.supports_endpoint(
-                    "hvac-status"
-                ),
+                "supports-hvac-status": vehicle_details.supports_endpoint("hvac-status"),
```

Comment:
Please revert this change, it is causing formatting issues.
```suggestion
                "supports-hvac-status": vehicle_details.supports_endpoint(
                    "hvac-status"
                ),
```